packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/index.js

```diff
@@ -71,12 +73,25 @@ export default function SidebarNavigationScreenNavigationMenus() {
 		getNavigationFallbackId();
 	}
 
-	const { handleSave, handleDelete, handleDuplicate } =
-		useNavigationMenuHandlers();
+	const history = useHistory();
 
 	const hasNavigationMenus = !! navigationMenus?.length;
 
-	if ( isLoading ) {
+	// If there is a **single** menu, then immediately
+	// redirect to the route for that menu. There are no
+	// dependencies for this effect because we always
+	// want it to run and it does no work if there is
+	// more than one menu.
+	useEffect( () => {
+		if ( navigationMenus?.length === 1 ) {
+			history.push( {
```

Comment:
We probably want `replace` here so that it doesn't add an additional stack to the history?